torch/utils/model_zoo.py

```diff
@@ -87,7 +87,8 @@ def _download_url_to_file(url, dst, hash_prefix):
                     break
                 f.write(buffer)
                 sha256.update(buffer)
-                pbar.update(len(buffer))
+                if progress:
+                    pbar.update(len(buffer))
```

Comment:
I think it will still print at least once due to the context manager. it would be nicer to make the manager a no-op if it's false